app/src/main/res/drawable/ic_keyboard_arrow_down_20dp.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Github needs a way to preview vector drawables. :D

Reply:
Can we somehow tag Chris Wanstrath here